lib/shopify-cli/oauth.rb

```diff
@@ -113,7 +113,7 @@ def listen_local
 
     def receive_access_code
       @access_code ||= begin
-        query = @server_thread.join(5).value
+        query = @server_thread.join(120).value
```

Comment:
this allows a 2 minute timeout which might need to be longer to accomodate slower people logging in.